bcda/api.go

```diff
@@ -361,6 +361,50 @@ func serveData(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, fmt.Sprintf("%s/%s/%s", dataDir, jobID, fileName))
 }
 
+/*
+	swagger:route POST /auth/token auth getAuthToken
+
+	Get access token
+
+	Verifies parameters client_id and secret, and returns a JWT token that can be presented to the other API endpoints.
+
+	Consumes:
+	- application/x-www-form-urlencoded
+
+	Produces:
+	- application/json
+
+	Schemes: https
+
```

Comment:
Please make swagger docs reflect basic auth header required for this endpoint. https://swagger.io/docs/specification/2-0/authentication/basic-authentication/

Reply:
@DeirdreHolub I've edited swagger docs to change to basic auth, but `swagger generate spec -o swaggerui/swagger.json -m` fails at the moment.  Passing code over to you.